### ISSUE TITLE
[Vote] Enable `k8s-metacollector` and `k8smeta` in the helm chart by default

### ISSUE THREAD

**Issue by incertum** [falcosecurity/falco#2995]
@falcosecurity/falco-maintainers 

re https://github.com/falcosecurity/falco/issues/2973#issuecomment-1877767412

Vote if the new k8smeta plugin should be enabled by default in the helm chart.

**Comment by incertum** [comment]
-1 (my vote) -> do not enable k8s-metacollector and k8smeta in the helm chart by default

Reasoning:

- Historically -k has caused a tremendous amount of issues and it wasn't used by Falco by default. We lack the data to ensure the new k8smeta plugin is more performant.
- However, more importantly the default Falco deployment does not use any of the new k8smeta fields, e.g. see the stable rules, they only use the k8s fields retrieved from the container runtime socket when running Falco with -pk (resolving the placeholder `%container.info` with the namespace and pod name).
- In general, I believe no plugin should be enabled by default in any templates or default configs we provide. Each plugin should be strictly opt-in. Because enabling one plugin versus disabling another plugin also begs the question as to why? What are the formal criteria to say a plugin should be enabled by default? ... Especially if not needed by the default Falco config it equates to wasted CPU cycles.

In summary, that way we ensure the default configs are the leanest possible and onboarding for newcomers has higher chances of going smooth and the debugging burden on maintainers when helping adopters will also be reduced.

**Comment by tspearconquest** [comment]
-1 also (Vote in favor of defaulting without -k) -- not a maintainer but I feel community votes should also be counted :)

**Comment by Andreagit97** [comment]
no strong opinions here, it's ok to disable it by default!

**Comment by FedeDP** [comment]
Agree, we can keep it disabled by default.

**Comment by leogr** [comment]
ok to keep it disabled for me.

**Comment by cccsss01** [comment]
disable for me

**Comment by Andreagit97** [comment]
This should be solved by [Falco 0.37.0](https://github.com/falcosecurity/falco/releases/tag/0.37.0)! `k8s-metacollector` and `k8smeta` are disabled by default in the helm chart
